### PR TITLE
SCALRCORE-27660 > TF registry > Provider Network Mirror Protocol

### PR DESCRIPTION
**Pull request by artemvang**
### Changelog
* [ ] I have updated CHANGELOG.md according to [CONTRIBUTING.md](../CONTRIBUTING.md)

### Documentation
* [ ] I have updated resource/data source documentation in [docs](../docs)

### State
* [ ] My changes affect the state
* [ ] I have included a state migration and a unit test for it

### CODE REVIEW

**Comment by maratkomarov** [comment]
@artemvang upload-dev свалился Error: Failed to update network mirror: Callback must be a function. Received undefined.

File: .github/workflows/upstream.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ on:
 
 jobs:
   acc-tests:
-    runs-on: ubuntu-latest
+    runs-on: [standart-2-amd64]
```
Comment (maratkomarov) [inline .github/workflows/upstream.yml:30]: @artemvang  typo: standar**d**

**Review by maratkomarov — CHANGES_REQUESTED** [review]

File: .github/workflows/upstream.yml (hunk 1 above)
Comment (artemvang) [reply .github/workflows/upstream.yml]: @maratkomarov all `runs-on` returned back to `ubuntu-latest`

Comment (maratkomarov) [reply .github/workflows/upstream.yml]: оке

**Review by maratkomarov — APPROVED** [review]

**Pull request merged**